src/main/java/org/generationcp/middleware/dao/oms/CVTermRelationshipDao.java

```diff
@@ -252,14 +254,15 @@ public Integer retrieveAnalysisDerivedVariableID(final Integer originalVariableI
 		}
 	}
 
-	public List<String> getCategoriesUsedInStudies(final int scaleId) {
+	public Set<String> getCategoriesInUse(final int scaleId) {
 		try {
-			final List<String> allCategories = new ArrayList<>();
+			final Set<String> allCategories = new HashSet<>();
 			allCategories.addAll(this.getScaleCategoriesUsedInObservations(scaleId));
 			allCategories.addAll(this.getScaleCategoriesUsedAsConditions(scaleId));
 			allCategories.addAll(this.getScaleCategoriesUsedAsGermplasmDescriptors(scaleId));
 			allCategories.addAll(this.getScaleCategoriesUsedAsTrialDesignFactors(scaleId));
 			allCategories.addAll(this.getScaleCategoriesUsedAsEnvironmentFactors(scaleId));
+			allCategories.addAll(this.getScaleCategoriesUsedInAttributes(scaleId));
 			return allCategories;
 		} catch (final HibernateException e) {
 			final String message = "Error in getCategoriesUsedInStudies in CVTermRelationshipDao: "
```

Comment:
Modify method in message as: `getCategoriesInUse`

Reply:
done.